src/main/java/org/terasology/math/delaunay/Poly2f.java

```diff
@@ -0,0 +1,198 @@
+// Copyright 2021 The Terasology Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+package org.terasology.math.delaunay;
+
+import com.google.common.base.Preconditions;
+import com.google.common.collect.ImmutableList;
+import org.joml.Vector2f;
+import org.joml.Vector2fc;
+import org.terasology.joml.geom.Rectanglef;
+import org.terasology.math.geom.Winding;
+
+import java.util.Collection;
+import java.util.List;
+
+public class Poly2f {
```

Comment:
I might be worth to move this to `joml-geom` at some point - ideally, when we know (or can think of) good use cases where this class would be used.

Would definitely put that on the backlog, though.